### PR TITLE
feat(extension): use proxy for token information

### PR DESCRIPTION
**Pull request by mchappell**
## Proposed solution

Move to use a proxy service for token information

## Testing

Lace token information functions as normal

---
# Allure report
`allure-report-publisher` generated test report!

**smokeTests**: ❌ [test report](https://lace-qa:8PP5wBaDV6UoXj@dq4ajm5i5q7bz.cloudfront.net/linux/chrome/2733/6665041807/index.html) for [eae64d95](https://github.com/input-output-hk/lace/pull/654/commits/eae64d95a26411730f6f321116b6de951efb075f)
|       | passed | failed | skipped | flaky | total | result |
|-------|--------|--------|---------|-------|-------|--------|
| Total | 31     | 1      | 0       | 0     | 32    | ❌     |

### CODE REVIEW

File: apps/browser-extension-wallet/src/lib/scripts/background/services/utilityServices.ts [hunk 1]
```diff
@@ -89,7 +89,7 @@ const handleChangeTheme = (data: ChangeThemeData) => requestMessage$.next({ type
 const { ADA_PRICE_CHECK_INTERVAL, SAVED_PRICE_DURATION } = config();
 const fetchTokenPrices = () => {
   // `base-policy-id=&base-tokenname=` for ADA as base token
-  fetch('https://api.muesliswap.com/list?base-policy-id=&base-tokenname=')
+  fetch('https://muesliswap.live-mainnet.eks.lw.iog.io/list?base-policy-id=&base-tokenname=')
```
[the remark this answers is not in the document]
Comment (pczeglik-iohk) [reply apps/browser-extension-wallet/src/lib/scripts/background/services/utilityServices.ts:92]: I'm not sure if I like `muesliswap` as a part of url (similar case to coingecko). Maybe we should create this proxy provider agnostic? 

And maybe we can extract this url to some better place, like maybe ENV var?

[the remark this answers is not in the document]
Comment (michalrus) [reply apps/browser-extension-wallet/src/lib/scripts/background/services/utilityServices.ts:92]: We do the same thing with CoinGecko, it’s a 420 kB JSON file served directly from RAM, I don’t think we will ever choke this server 😊

```
❯ curl -sI 'https://muesliswap.live-mainnet.eks.lw.iog.io/list?base-policy-id=&base-tokenname='  | grep length
content-length: 432030
```

Both CoinGecko and this one run the same exact code on `live-mainnet` and preview environments.

[the remark this answers is not in the document]
Comment (pczeglik-iohk) [reply apps/browser-extension-wallet/src/lib/scripts/background/services/utilityServices.ts:92]: Ok, I got it @michalrus.

In terms of e2e test what I see [here](https://github.com/input-output-hk/lace/actions/runs/6588507322/job/17900913231?pr=654) is that our tests are executed against `preprod`

```
[0-78] available networks: (preview|preprod|mainnet), falling back to: preprod
[0-78] Using network: Preprod with id: 1
```

I ask as maybe it should be aligned?

btw. @ljagiela it looks like we are not setting `ENV` var to select network, that's why it falls back to `preprod`, right?

Can we maybe set it explicit, so there is no logs produced which makes a noise in output logs?

[the remark this answers is not in the document]
Comment (mchappell) [reply apps/browser-extension-wallet/src/lib/scripts/background/services/utilityServices.ts:92]: @pczeglik-iohk I think the requested changes here lie outside of the scope of this PR. Please create a ticket detailing your proposed changes and we can look at this in a future improvement. If this satisfies your needs please resolve this conversation.

**Review by wklos-iohk — APPROVED** [review]

**Pull request merged**